base/setup/lib/setuplib.h

```diff
@@ -40,6 +40,17 @@ extern HANDLE ProcessHeap;
 #include "utils/arcname.h"
 #include "utils/osdetect.h"
 #include "utils/regutil.h"
+
+typedef enum _ARCHITECTURE_TYPE
+{
+    ARCH_PcAT,      //< Standard BIOS-based PC-AT
+    ARCH_NEC98x86,  //< NEC PC-98
+    ARCH_Xbox,      //< Original Xbox
+    ARCH_Arc,       //< ARC-based (MIPS, SGI)
+    ARCH_Efi,       //< EFI and UEFI
+// Place other architectures supported by the Setup below.
+} ARCHITECTURE_TYPE;
+
```

Comment:
Was it neccessary to move this upwards to make it compile? Or could it be kept, where it was located before at line 69?

Reply:
Yes it was